oracle/types/market_test.go

```diff
@@ -26,23 +26,32 @@ func TestProviderTickersFromMarketMap(t *testing.T) {
 			err:      false,
 		},
 		{
-			name:     "invalid market map",
+			name:     "single disabled market",
```

Comment:
I think the thing which is missing here for me is:
* what do we do if there is a transitive dependency on a disabled market?
For example, do we allow `USDT/USDC` to be disabled if that's our normalize market for other things?

Reply:
I was thinking the same thing, if a given feed should be accessible but the overall market shouldn't be calculated. Is this a case we should be worried about?